scripts/hudson/narayana.sh

```diff
@@ -986,8 +971,13 @@ export ANT_OPTS="$ANT_OPTS $IPV6_OPTS"
 # run the job
 
 [ $NARAYANA_BUILD = 1 ] && build_narayana "$@"
-[ $AS_CLONE = 1 ] && clone_as "$@"
-[ $AS_BUILD = 1 ] && build_as "$@"
+
+if [ "$_jdk" -ge 17 ]; then
+  [ $AS_BUILD = 1 ] && AS_DOWNLOAD=1 # 26.x branch won't build on JDK 17 so download it instead
```

Comment:
Using a downloaded AS will just run the tests using the Narayana that is already in there rather than the Narayana that we are building so won't verify that changes in Narayana have not broken anything.